.github/workflows/ci.yml

```diff
@@ -0,0 +1,94 @@
+name: SROS2 CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 12 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+    - uses: ros-tooling/setup-ros@0.0.13
+    - uses: ros-tooling/action-ros-ci@0.0.13
+      with:
+        package-name: |
+          sros2
+          sros2_cmake
+    - name: Upload Logs
+      uses: actions/upload-artifact@v1
+      if: failure()
+      with:
+        name: colcon-logs
+        path: ros_ws/log
+
+  # Linter jobs
+  ament_copyright:
```

Comment:
Re-thinking this, is it actually necessary to have this given that we also have [this](https://github.com/ros2/sros2/blob/master/sros2/test/test_copyright.py)? Doesn't it mean that we're running it as part of the test job anyway?

Reply:
>  Doesn't it mean that we're running it as part of the test job anyway?

Yes we are

The main advantage I see for this is a faster feedback loop, these linter jobs will give result ~10times faster than the test job or the buildfarm.
What seems to be the most commonly overlooked parts in contributions are usually style (and coverage). Being able to give fast feedback on those can help reduce iteration loop time and the number of iterations. 

So I tend towards keeping those for now and consider removing them if it proves to not be helpful or give results not matching the ones from the buildfarm

Reply:
Fair point, as long as we realize:

- We're running some of them twice (once in the linter job, once in the test job)
- We're running a different set of linters in the build farm due to the fact that we don't have colcon tests for all of the ones here